thirdparty/cmdconfig/commands/fnevalcmd_test.go

```diff
@@ -293,7 +221,7 @@ apiVersion: v1
 		},
 		{
 			name: "envs",
-			args: []string{"run", "dir", "--env", "FOO=BAR", "-e", "BAR"},
+			args: []string{"eval", "dir", "--env", "FOO=BAR", "-e", "BAR"},
```

Comment:
Is declarative run being removed in this PR?

Also please updating the command reference as well as part of this issue (can be separate PR).

Reply:
> Is declarative run being removed in this PR?
No, this PR only removes the flags. Separate PR for removing declarative.
> Also please updating the command reference as well as part of this issue (can be separate PR).
Will do.